mypyc/build.py

```diff
@@ -431,7 +431,8 @@ def mypycify(
     *,
     only_compile_paths: Optional[Iterable[str]] = None,
     verbose: bool = False,
-    opt_level: str = '3',
+    opt_level: str = "3",
+    debug_level: str = "1",
```

Comment:
please also describe the arg in the following docstring

Reply:
Done!